browser_tests/tests/vueNodes/interactions/links/linkInteraction.spec.ts

```
@@ -828,7 +828,7 @@ test.describe('Vue Node Link Interaction', () => {
   })
 
   test.describe('Release actions (Shift-drop)', () => {
-    test('Context menu opens and endpoint is pinned on Shift-drop', async ({
+    test.fixme('Context menu opens and endpoint is pinned on Shift-drop', async ({
       comfyPage,
       comfyMouse
     }) => {
```

ESLint

[error] Replace `'Context·menu·opens·and·endpoint·is·pinned·on·Shift-drop',·async·({⏎······comfyPage,⏎······comfyMouse⏎···` with `⏎······'Context·menu·opens·and·endpoint·is·pinned·on·Shift-drop',⏎······async·({·comfyPage,·comfyMouse`

(prettier/prettier)